content/rs/databases/active-active/get-started-active-active.md

```diff
@@ -1,29 +1,34 @@
 ---
-Title: Getting Started with Redis Enterprise Active-Active Databases
-linktitle: Active-Active databases
-description:
-weight: 30
+Title: Get Started with Redis Enterprise Active-Active databases
+linktitle: Get started with Active-Active
+description: Quick start guide to create an Active-Active database for test and development.
+weight: 20
 alwaysopen: false
-aliases: /rs/getting-started/creating-database/crdbs/
-         /rs/getting-started/getting-started-crdbs/
+aliases: [
+    /rs/getting-started/creating-database/crdbs/,
+    /rs/getting-started/getting-started-crdbs/,
+    /rs/databases/active-active/get-started-active-active.md,
+    /rs/databases/active-active/get-started-active-active/,
+
+]
 categories: ["RS"]
 ---
-In this guide, we'll set up an Active-Active database, formerly known as CRDB (Conflict-free
-Replicated DataBase) spanning across two Redis Enterprise Software
+
+To get you started, this article will help you set up a Active-Active database, formerly known as CRDB (conflict-free replicated database) spanning across two Redis Enterprise Software
 clusters for test and development environments. Here are the steps:
 
 - Step 1: Run two Redis Enterprise Software (RS) Docker containers
 - Step 2: Set up each container as a cluster
 - Step 3: Create a new Redis Enterprise Active-Active database
 - Step 4: Test connectivity to the Active-Active database
 
-To run an Active-Active database on installations from the [RS download package]({{< relref "/rs/getting-started/_index.md" >}}),
+To run an Active-Active database on installations from the [RS download package]({{< relref "rs/installing-upgrading/get-started-redis-enterprise-software.md" >}}),
 set up two RS installations and continue from Step 2.
 
 {{< note >}}
 This getting started guide is for development or demonstration environments.
 To set up an Active-Active database in a production environment, use the instructions for
-[creating an Active-Active database]({{< relref "/rs/administering/creating-databases/create-active-active.md" >}}).
+[creating an Active-Active database]({{< relref "/rs/databases/active-active/create-active-active.md" >}}).
 {{< /note >}}
 
```

Comment:
Maybe change the following headers:
- "Connecting using redis-cli" to "Connect with redis-cli"
- "Connecting using Hello World application in Python"  to "Connect with Python"

There's also some instances where Redis and Python are not capitalized properly.